src/main/java/org/helpberkeley/memberdata/HBParser.java

```diff
@@ -713,9 +681,7 @@ static RestaurantTemplatePost restaurantTemplatePost(final String rawPost) {
 
         for (String line : rawPost.split("\n")) {
             if (line.contains(Constants.UPLOAD_URI_PREFIX)) {
```

Comment:
This is a spot where we are also exposed to alternate upload URLs.   Add boolean static on UploadFile that checks for both and use it here.